coalib/output/Logging.py

```diff
@@ -1,4 +1,5 @@
 from datetime import datetime
+import sys
 import json
 import io
 import logging
```

Comment:
I'm not sure I understand.
Why does "io" occur after "json" and before "logging"?